### DISCUSSION TITLE
Links to headers in other file doesn't work

### DISCUSSION THREAD

**Discussion by lua-rocks** [gsantner/markor#1782] [Q&A] [answered]
Links to headers in current file works fine:

```
[Link](#header)
```

Links to other files also works:

```
[Link](file.md)
```

But links to headers in other files doesn't works:

```
[Link](file.md#header)
```

**Comment by gsantner** [comment, marked as answer]
Well, ""doesn't work"" is relative, it's not implemented. What doesn't exist can't be broken :-).  
This is a feature request, is what I want to say, not a bug.

Feel free to make improvements by opening merge request(s).
